### ISSUE TITLE
Dead?

### ISSUE THREAD

**Issue by sabond** [meteor-vue/vue-meteor#411]
Is this project dead?

**Comment by chris-visser** [comment]
It is very alive. The Vue guide is currently being rewritten. There's a new integration package coming to connect to other Vue frameworks and there's a Vue 3 updating planned as soon as it goes into stable mode

**Comment by 99cv** [comment]
@chris-visser 
What exactly do you mean with other vue frameworks? Something like quasar?

Is there any rough timeline for the "vue3 updating"?

I appreciate keeping it alive! ;-)

**Comment by chris-visser** [comment]
Yes. Planning to start with the update coming week. I'm not sure how long it will take. I need to familiarize myself with some of the build steps and Vue 3. I will give you an update next week

**Comment by 99cv** [comment]
Hi @chris-visser ,
I just wanted to ask you if you were able to make any progress with the updating project?

**Comment by chris-visser** [comment]
Hey. Yes well.. progress as in figuring out how to build Vue3 and how to combine it with Meteor. Still have to do it now. Bit in doubt if I need to wait until Meteor 2.0 (has HMR) and it would save time to leverage that in stead of a custom HMR.

**Comment by quank123wip** [comment]
It has been a year since the last reply... Is this project dead? If it isn't, I can't wait to use Vue 3.0 and Vuetify 3.0 with Meteor this year.
